src/python/pants/backend/helm/goals/tailor.py

```diff
@@ -49,7 +49,7 @@ async def find_putative_helm_targets(
         putative_targets.append(
             PutativeTarget.for_target_type(
                 HelmChartTarget,
-                name=os.path.basename(dirname),
+                name="chart",
```

Comment:
This name is not temporary, per the proposal at https://docs.google.com/document/d/1N1bX2PsEHL-Ue9_Spsl3NlyHxmgKtvpB5y4RAD-cOZQ/edit#. A `helm_chart` is an "atom target" so the new shortcut for generated targets won't apply.

The address will look like `path/to:chart`.